### ISSUE TITLE
ThunderWallet creates new testnet address every time it is run (forgets old one)

### ISSUE THREAD

**Issue by lacksfish** [blockchain/thunder#26]
Would be great to have a permanent storage (wallet) for testnet private keys. Am I missing something?

**Comment by lacksfish** [comment]
Also, restore from words prompts me that there is 1 tBTC sitting in the current Wallet and that I should empty it. Can not proceed testing this way.

**Comment by matsjj** [comment]
Yes, so by default it is not connecting to any blockchain, it is using the MockWallet object. This allows for easier testing and debugging, so you won't see any coins sent to the wallet (because it's not listening the blockchain). 

If you want to try it out with real tBTC, you have to toggle Constants.USE_MOCK_BLOCKCHAIN. Then it will create a wallet file and actually read from it upon startup again. Keep in mind that there is no persistence data storage engine yet (working on that just now).
